assets/csidriveroperators/shared-resource/12_servicemonitor.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: shared-resource-csi-driver-node
```

Comment:
yeah this needs to be https://github.com/openshift/csi-driver-shared-resource-operator/blob/master/manifests/10_servicemonitor.yaml#L5 


we have metric endpoints 


the one from https://github.com/openshift/csi-driver-shared-resource-operator/blob/master/manifests/10_servicemonitor.yaml#L5 is suppose to be the one for the operator's metrics endpoint

the `shared-resource-csi-driver-node` is from https://github.com/openshift/csi-driver-shared-resource-operator/blob/master/assets/servicemonitor.yaml#L5 and is for the driver's metric endpoint ... that one says in the shared resource operator assets folder

Now, if you and @alicerum want to work on the names a bit to make sure it is clean while I'm out tomorrow, to make sure we clearly deliniate operator vs. driver here, go for it